app/src/js/utils/type.js

```diff
@@ -17,20 +17,15 @@ export const reconciliationReportTypes = [
 ];
 
 export const operationTypes = [
-  {
-    id: 'Bulk Granules',
-    label: 'Bulk Granules',
-  },
-  {
-    id: 'ES Index',
-    label: 'ES Index',
-  },
-  {
-    id: 'Bulk Delete',
-    label: 'Bulk Delete',
-  },
-  {
-    id: 'Kinesis Replay',
-    label: 'Kinesis Replay',
-  },
-];
+  'Bulk Granule Delete',
+  'Bulk Granule Reingest',
+  'Bulk Granules',
+  'Bulk Execution Delete',
+  'Data Migration',
+  'Dead-Letter Processing',
+  'DLA Migration',
+  'ES Index',
+  'Kinesis Replay',
+  'Reconciliation Report',
+  'SQS Replay',
+].map((name) => ({ id: name, label: name }));
```

Comment:
Presumably the original thinking was that the id and label would/could be different. If we're just defining what the ID is here, yeah. No reason to not make it the same.

Reply:
I added some missing types.